minisite.install

```diff
@@ -5,182 +5,57 @@
  * Contains install and update functions for Minisite.
  */
 
-/**
- * Implements hook_install().
- */
-function minisite_install() {
-  // Create the minisite directory and ensure it's writable.
-  $directory = file_default_scheme() . '://minisite';
-  file_prepare_directory($directory, FILE_CREATE_DIRECTORY | FILE_MODIFY_PERMISSIONS);
-}
+use Drupal\Core\File\FileSystemInterface;
+use Drupal\minisite\Minisite;
 
 /**
  * Implements hook_uninstall().
  */
 function minisite_uninstall() {
   // Remove the minisite directory and generated files.
-  file_unmanaged_delete_recursive(file_default_scheme() . '://minisite');
+  \Drupal::service('file_system')->deleteRecursive(Minisite::getCommonArchiveDir());
+  \Drupal::service('file_system')->deleteRecursive(Minisite::getCommonAssetDir());
 }
 
 /**
  * Implements hook_requirements().
  */
 function minisite_requirements($phase) {
   if ($phase != 'runtime') {
-    return array();
+    return [];
   }
 
-  $requirements = array();
+  $requirements = [];
 
   if ($phase == 'runtime') {
-    $requirements['minisite_upload'] = array(
-      'title' => t('Minisite upload directory'),
+    $path = Minisite::getCommonArchiveDir();
```

Comment:
mainly moved code around to allow paths in Status Report. Like so:
<img width="516" alt="Status_report___Drush_Site-Install" src="https://user-images.githubusercontent.com/378794/69198153-e6451900-0b87-11ea-9b8b-b8fdcd1c0487.png">